build/images/flow-aggregator/Dockerfile.coverage

```diff
@@ -29,10 +29,15 @@ LABEL description="The docker image for the flow aggregator with code coverage m
 
 USER root
 
-COPY --from=flow-aggregator-build /antrea/bin/flow-aggregator* /usr/local/bin/
-COPY --from=flow-aggregator-build /antrea/test/e2e/coverage/flow-aggregator-arg-file /
-COPY --from=flow-aggregator-build /antrea/bin/antctl* /usr/local/bin/
+COPY --from=flow-aggregator-build /antrea/bin/flow-aggregator /usr/local/bin/
+COPY --from=flow-aggregator-build /antrea/bin/flow-aggregator-coverage /usr/local/bin/
+COPY --from=flow-aggregator-build /antrea/bin/antctl /usr/local/bin/
+COPY --from=flow-aggregator-build /antrea/bin/antctl-coverage /usr/local/bin/
```

Comment:
Why to change this? The old one can't work anymore?

Reply:
Don't exactly remember why I changed this. Should not make any difference. Will revert.